mlflow/pyfunc/__init__.py

```diff
@@ -828,7 +852,13 @@ def _create_model_downloading_tmp_dir(should_use_nfs):
 _MLFLOW_SERVER_OUTPUT_TAIL_LINES_TO_KEEP = 200
 
 
-def spark_udf(spark, model_uri, result_type="double", env_manager=_EnvManager.LOCAL):
+def spark_udf(
+    spark,
+    model_uri,
+    result_type="double",
+    env_manager=_EnvManager.LOCAL,
+    params: Optional[Dict[str, Any]] = None,
+):
```

Comment:
Can we add a test case verifyin that the parameters get passed to the pyfunc model in the UDF correctly? I see that we added a test case for sentence transformers + spark UDF, but that test case doesn't verify that the params are passed correctly through to the model

I think it would be better to add a specific test case for this to `test_spark.py`